CHANGELOG.md

```diff
@@ -34,6 +35,100 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v1.1.0-rc1
+
+We expect that this release candidate is quite close to the final v1.1.0
+release. However, breaking API changes are still possible.
+
+This release candidate is suitable for implementors, but we do not recommend
+shipping products based on a release candidate API due to the possibility of
+incompatible changes prior to the final release. The following represents the
+changes since v1.0.0:
+
+## Standard Channel
+
+### GRPCRoute has Graduated to GA 🎉
+
+GRPCRoute has graduated to GA (v1) and is now part of the Standard Channel. If
+you are already using the experimental version GRPCRoute, we recommend holding
+off on upgrading to the standard channel version of GRPCRoute until the
+controllers you're using have been updated to support GRPCRoute v1. Until then,
+it is safe to upgrade to the experimental channel version of GRPCRoute in v1.1
+that includes both v1alpha2 and v1 API versions.
+
+Leading Contributor: @gnossen
+
+### Mesh Support
+
+The standard for using Gateway API for Mesh has formally graduated to v1.0.
+
+Leading Contributors: @howardjohn, @keithmattix, @kflynn
+
+### Conformance Profiles and Reports
+
+The concept of conformance profiles and the corresponding reports have graduated
+to GA.
+
+Leading Contributors: @mlavacca, @shaneutt
+
+### Port in ParentRefs
+
+You can now use a `port` field in ParentRefs to attach to Gateways, Services, or
+other parent resources.
+
+Leading Contributor: @frankbu
+
+## Experimental Channel
+
+### Session Persistence + BackendLBPolicy
+Session Persistence is being introduced to Gateway API via a new policy
+(BackendLBPolicy) and Route-level configuration for both HTTPRoute and
+GRPCRoute.
+
+Leading Contributors: @gcs278, @ginayeh
```

Comment:
@gcs278 @ginayeh can you help fill this out?

Reply:
This seems reasonable as a high-level release note. Do you expect we should go into more detail? I don't have any in particular I think we should say, expect maybe what fields are being introduced (SessionName, Timeouts, Type...), but that might be too much for this release note based on the other examples I see.

Experimental PR has been created https://github.com/kubernetes-sigs/gateway-api/pull/2969.

Reply:
Okay - I came up with a bit more detail, please tweak as you see fit. I can provide more detail if needed.

```suggestion
Session Persistence is being introduced to Gateway API via a new policy
(BackendLBPolicy) for Service-level configuration and as fields within HTTPRoute and
GRPCRoute for Route-level configuration. The BackendLBPolicy and Route-level
APIs provide the same session persistence configuration, including session
timeouts, session name, session type, and cookie lifetime type.

Leading Contributors: @gcs278, @ginayeh
```